parsl/executors/high_throughput/executor.py

```diff
@@ -258,6 +270,11 @@ def __init__(self,
         self.worker_logdir_root = worker_logdir_root
         self.cpu_affinity = cpu_affinity
 
+        self.enable_mpi_mode = enable_mpi_mode
+        assert mpi_launcher in VALID_LAUNCHERS, \
```

Comment:
I think if you do a `Literal` sequence in the type specification, typeguard might be able to check this

Reply:
probably fine to leave it like this then: it's only worth doing if it makes things easier, I think.